access-checkout-react-native-sdk/src/hooks/useCardConfig.ts

```diff
@@ -0,0 +1,32 @@
+import type { CardValidationEventListener } from '../validation/CardValidationEventListener';
+import { CardValidationConfig } from '../validation/CardValidationConfig';
+import CardConfig from '../config/CardConfig';
+
+export interface UseCardConfig {
+  panId: string;
+  expiryDateId: string;
+  cvcId: string;
+  validationConfig?: {
+    acceptedCardBrands?: string[];
+    enablePanFormatting?: boolean;
+    validationListener: CardValidationEventListener;
```

Comment:
Can we possibly strongly type validationConfig so that it's of typr `CardValidationConfig` and change `CardValidationConfig` to be an interface?

Reply:
As agreed. in offline discussion we will change the validationConfig to be of type CardValidationConfig (using interfaces) and we will rename our internal class CardValidationConfig to MerchantCardValidationConfig